storage/storage.go

```diff
@@ -1297,6 +1297,18 @@ type Query struct {
 	// the query. It's used internally and is populated for the user by
 	// calling Query.SetAttrSelection
 	fieldSelection string
+
+	// StartOffset sets the optional parameter "startOffset": Filter results
```

Comment:
No need for the "sets the optional parameter" verbiage. I would say:

`StartOffset is used to filter results to objects whose names are...` etc.

Reply:
> I think an integration test might make sense for this feature-- something like
> 
> https://github.com/googleapis/google-cloud-go/blob/2a870c27ead0e5f70b8290079fce1e5005e4f427/storage/integration_test.go#L1055
> 
> and the tests below that. @AlisskaPie could you create this?

Yes, I'll do this.
Thank you for the detailed answer!

Reply:
Done